test/mocha/20-validate-config.js

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2017 Digital Bazaar, Inc. All rights reserved.
+ */
+/* globals should */
+
+'use strict';
+
+const bedrock = require('bedrock');
+const brValidator = require('bedrock-ledger-validator-equihash');
+
+const mockData = require('./mock.data');
+
+describe('validateConfiguration API', () => {
+  it('validates a proper config', done => {
+    const testConfig = mockData.ledgers.alpha.config.input[0].eventValidator[0];
+    brValidator.validateConfiguration(testConfig, err => {
+      should.not.exist(err);
```

Comment:
I did some experimenting with this idea after we spoke about it.  I found that calling `done(err)` does not give you a stack trace, so one ends up having to add logging to figure out what's going on anyway.  Did you have a different experience?